guides/policy/azure-authenticator/index.html

```diff
@@ -0,0 +1,140 @@
+      - !host
+        id: {{app ID}}
+        annotations:
+          authn-azure/subscription-id: {{subscription ID}}
+          authn-azure/resource-group: {{resource group}}
+          authn-azure/{{user or system}}-assigned-identity: {{identity}}
+</pre>
+
+        <h3>Policy</h3>
+        <pre class="container border" id="output">
+</pre>
+
+    </div>
+</div>
+
+<script>
+
+    $(document).ready(function () {
+
+        $('#encodedToken').on('input', function () {
+            updateCmd();
+        });
+
+        $('#tblInput input').on('input', function () {
+            updateCmd();
+        });
+    });
+
+    function updateCmd() {
+        let output = "";
+        let appId = $("#txtAppId").val();
+        let encodedToken = $("#encodedToken").val();
+        if (!appId){
+            output = "Application ID is missing";
+        }
+        else if (!encodedToken){
+            output = "Azure token is missing";
+        }
+        else {
+            output = getPolicyFromToken(encodedToken, appId);
+        }
+        $("#output").html(output);
+    }
+
+    function getPolicyFromToken(encodedToken, appId) {
+        try {
+            let policy = $("#theTemplate").html();
+            let parsedToken = parseJwt(encodedToken);
+            let token = parseAzureToken(parsedToken);
+
+            policy = policy.replace("{{app ID}}", appId);
+            policy = policy.replace("{{subscription ID}}", token.subscriptionId);
+            policy = policy.replace("{{resource group}}", token.resourceGroup);
+            policy = policy.replace("{{user or system}}", token.isUserIdentity ? "user" : "system");
+            policy = policy.replace("{{identity}}", token.identity);
+            return policy;
+        }
+        catch (e) {
+            console.error(e);
+            return "Failed to parse Azure Token"
+        }
+    }
+
+    function parseJwt (token) {
+        var base64Url = token.split('.')[1];
+        var base64 = base64Url.replace(/-/g, '+').replace(/_/g, '/');
+        var jsonPayload = decodeURIComponent(atob(base64).split('').map(function(c) {
+            return '%' + ('00' + c.charCodeAt(0).toString(16)).slice(-2);
+        }).join(''));
+
+        return JSON.parse(jsonPayload);
+    }
+
+    function parseAzureToken (token) {
+        var azureIdentity = {};
+
+        var xmsMirid = token.xms_mirid.split('/');
+        for (i = 0; i < xmsMirid.length; i++){
+            switch (xmsMirid[i]) {
+                case "subscriptions":
+                    azureIdentity.subscriptionId = xmsMirid[i + 1];
+                    i++;
+                    break;
+                case "resourcegroups":
+                    azureIdentity.resourceGroup = xmsMirid[i + 1];
+                    i++;
+                    break;
+                case "providers":
+                    azureIdentity.isUserIdentity = xmsMirid[i + 1] === "Microsoft.ManagedIdentity";
```

Comment:
When developing, we were hesitant to relay on length, indices, etc because if Azure decided to add another property, that would break our design. So I advise you have the same hesitation. 

Take a look how we handled it on the server side. We did something like the following:
``` 
if xms_mirid.providers.include? "Microsoft.ManagedIdentity"
          @token_identity[:user_assigned_identity] = xms_mirid.providers.last
.
.
.
```
So we are checking if `Microsoft.ManagedIdentity` exists anywhere in the providers sub array